.github/workflows/push.yml

```diff
@@ -1,4 +1,4 @@
-name: Push
+name: CI
```

Comment:
Let's make this more even more descriptive, maybe "Container Image Push"?

Reply:
This is the text that shows up on the badge. It currently says "Push". I though "CI" would be a better name for the badge.